src/components/foodDescription/components/foodListItem/FoodListItem.js

```diff
@@ -0,0 +1,66 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+// readers
+import foodListItemReader from "../../readers/foodListItemReader";
+
+// constants
+import currencySymbol from "../../../../constants/currency";
+
+// icons
+import vegSymbol from '../../../../assets/images/vegSymbol.svg'
+
+// css
+import './foodListItem.css'
+
+function FoodListItem(props) {
+
+    const { item } = props;
+
+    const name = foodListItemReader.name(item);
+    const amount = foodListItemReader.amount(item);
+    const description = foodListItemReader.description(item);
+    const type = foodListItemReader.type(item);
+    const currency = foodListItemReader.currency(item);
+
+    return (
+        <>
+            <div className="food-list-item">
+                <img src={vegSymbol} className={`m-tb-0-5 h-1 ${type}` } alt={type}/>
```

Comment:
check this once, you're always showing vegSymbol and why the type is in className ??